vllmd-hypervisor/preseed-v1-bookworm.cfg

```diff
@@ -0,0 +1,183 @@
+d-i partman/confirm_nooverwrite boolean true
+
+# Select disk /dev/vdc
+d-i partman/early_command string debconf-set partman-auto/disk /dev/vdc
+d-i partman-auto/disk string /dev/vdc
+d-i partman-auto/method string regular
+d-i partman/default_disk string /dev/vdc
+
+# Or specify a custom partitioning recipe
+# d-i partman-auto/expert_recipe string ...
+
+# Accept the partition layout without confirmation
+d-i partman-partitioning/confirm_write_new_label boolean true
+
+
+####
+## B.4.7.aw1 Boot defaults
+
+d-i partman-efi/non_efi_system boolean true
+d-i partman-partitioning/choose_label select gpt
+d-i partman-partitioning/default_label string gpt
+
+
+####
+## B.4.7.3 Controlling how partitions are mounted
+
+d-i partman/mount_style select uuid
+
+
+####
+## B.4.8 Base system installation
+
+d-i base-installer/install-recommends boolean true
+d-i base-installer/initramfs-tools/driver-policy select generic
+d-i base-installer/kernel/image select linux-image-amd64
+
+
+####
+## B.4.9 Apt setup
+
+#d-i apt-setup/cdrom/set-first boolean false
+d-i apt-setup/non-free-firmware boolean false
+d-i apt-setup/non-free boolean false
+d-i apt-setup/contrib boolean false
+d-i apt-setup/disable-cdrom-entries boolean true
+d-i apt-setup/services-select multiselect security, updates
+d-i apt-setup/security_host string security.debian.org
+d-i debian-installer/allow_unauthenticated boolean false
+
+
+####
+## B.4.10 Package selection
+
+tasksel tasksel/first multiselect standard, ssh-server
+d-i pkgsel/include string sudo cloud-init curl open-iscsi libopeniscsiusr openssh-server neovim ssh-import-id
+d-i pkgsel/upgrade select full-upgrade
+popularity-contest popularity-contest/participate boolean false
+
+
+####
+## B.4.11 Boot loader installation
+
+d-i grub-installer/only_debian boolean true
+d-i grub-installer/grub2_instead_of_grub_legacy boolean true
+#d-i grub-installer/bootdev string /dev/sda
+d-i grub-pc/mixed_legacy_and_grub2 boolean false
+d-i grub-pc/chainload_from_menu.lst boolean false
+d-i grub2/linux_cmdline string console=tty1 console=ttyS0,115200
+
+# d-i grub2/linux_cmdline string <string>
+# # d-i grub2/linux_cmdline_default string quiet
+#
+#d-i debian-installer/add-kernel-opts console=tty1 console=ttyS0,115200
+#d-i grub-pc/install_devices multiselect /dev/vda
+
+
+# NB: https://wiki.debian.org/UEFI#Force_grub-efi_installation_to_the_removable_media_path
+#d-i grub-installer/force-efi-extra-removable boolean true
+
+# d-i grub2/force_efi_extra_removable boolean false
+
+
+# d-i finish-install/reboot_in_progress note
+#
+# reviewed up to: libapache-sessionx-perl_2.01-5_all.deb
+############################
+
+d-i preseed/late_command string \
+    echo GRUB_SERIAL_COMMAND="serial --unit=0 --speed=115200" >> /target/etc/default/grub/grub.d/serial_command.cfg; \
+    echo GRUB_TERMINAL="serial console" >> /target/etc/default/grub/grub.d/terminal.cfg; \
+    echo "packer ALL=(ALL:ALL) NOPASSWD:ALL" > /target/etc/sudoers.d/packer; \
```

Comment:
the only acceptable sudo user is sdake. delete any remann ts of the word packer.